### DISCUSSION TITLE
Installing python-lsp-server via mason - jedi missing?

### DISCUSSION THREAD

**Discussion by blumm94** [mason-org/mason.nvim#1397] [Q&A]
Hi,

I am using neovim 0.9.1 on WSL2 (Debian 12). I am also using the default configuration of mason and mason-lspconfig. As LSP server, I've installed python-lsp-server via mason. Unfortunately, most features that require jedi (the pylsp docu says that the LSP depends on jedi) are not working. My assumption is that mason does not automatically install jedi. 

Is this true? Do I have to install jedi manually?

Some other dependencies are automatically installed under ~/.local/share/nvim/mason/packages/python-lsp-server/venv/bin. For example pylint or flake8.

The logs do not show any errors. 

I am new to neovim and struggle with the configuration, so if I have missed to provide necessary information, please let me know :)

Thank you!
Regards,
Martin

**Comment by blumm94** [comment]
ok nevermind... I found it...

**Reply by mmoya** [reply]
Hi @blumm94 

I have the same problem but I don't have `jedi` in `~/.local/share/nvim/mason/packages/python-lsp-server/venv/bin`. Can you clarify what you found and where?

Thanks
